tests/framework/src/backends/navigator.rs

```diff
@@ -51,6 +46,16 @@ impl SuccessResponse for TestResponse {
         self.redirected
     }
 
+    fn next_chunk(&mut self) -> OwnedFuture<Option<Vec<u8>>, Error> {
```

Comment:
Can you please drag the code back to original place, before `fn status`? :D

Reply:
Why? This way, the implementation order is the same as the trait order (and the implementation order in every other implementation). It's cleaner to have it consistent (and shows up as warning otherwise in some IDEs).

Reply:
The disadvantage of moving code around is that its blame history is lost (it's harder to track changes). IIRC there are more places where the order is not preserved.

So I guess it's a decision of what's better in this case (preserving the order vs preserving blame history).

Maybe it's also worth considering adding a lint for that when we decide that preserving the order is important.

Reply:
Hmm, I see. I get that point, but personally, I think code is slightly changed all the time, especially in a project that moves as fast as Ruffle. When you use blame, you often need to look at the changes *before* the last one as refactors and other minor changes are frequent, so changing the order to improve code quality doesn't disrupt workflows.
There can be reasons to purposely have a different order in some places, but I think it's clean to generally try to preserve it.